tasks/main.yml

```diff
@@ -225,6 +225,7 @@
             -in {{ ca_client_ca_dir }}/{{ inventory_hostname }}.key
             -topk8
             -passin pass:{{ ca_keypassphrase | default(omit, true) }}
+            {% if ca_ls7_workaround | bool %}-v1 PBE-SHA1-RC4-128{% endif %}
```

Comment:
How about putting the Cipher in a default variable? So the user can change it as he like.
`{% if ca_ls7_workaround | bool %}-v1 {{ ca_ls7_cipher }}{% endif %}`

Reply:
The ca_ls7_cipher should be defined under defaults.yml

Reply:
Great idea! I'll implement it